src/common/components/first-time-heading/index.js

```diff
@@ -91,7 +91,7 @@ FirstTimeHeading.propTypes = {
 
 function mapStateToProps(state) {
   return {
-    snapsLoaded: state.snaps.success,
+    snapsLoaded: !!state.snaps,
```

Comment:
Don't rely on `success` flag, as it makes first time progress blink on polling. Fixes #499